openapi/src/main/resources/templates/openworld-sdk/fieldConstraintViolation.mustache

```diff
@@ -0,0 +1,7 @@
+package {{packageName}}.models.exception
+
+data class FieldConstraintViolation(
```

Comment:
1. Can we please document this (and the other) new classes?
2. Let's rename It to `PropertyConstraintViolation`. The same applies to `FieldConstraintViolationException` --> `PropertyConstraintViolationException`.